python-flask/.env

```diff
@@ -0,0 +1,7 @@
+# for local development define POSTGRES_HOST with command `ipconfig getifaddr en0`
+POSTGRES_USER=user
```

Comment:
I would rather create only one env variable like DB_URL and remove all the other 5 variables